docs/quick-start.md

```diff
@@ -3,22 +3,374 @@ title: Getting Started
+automatically satisfy resource claims.
+
+Here's a resource class that will dynamically provision Cloud SQL instances with
+the same settings as the `CloudSqlInstance` we provisioned earlier in the guide:
+
+```yaml
+apiVersion: database.gcp.crossplane.io/v1beta1
+kind: CloudsqlInstanceClass
+metadata:
+  name: example-cloudsql-class
+  annotations:
+    resourceclass.crossplane.io/is-default-class: "true"
+  labels:
+    guide: getting-started
+specTemplate:
+  providerRef:
+    name: example
+  writeConnectionSecretsToNamespace: crossplane-system
+  forProvider:
+    databaseVersion: MYSQL_5_6
+    region: us-west2
+    settings:
+      tier: db-n1-standard-1
+      dataDiskType: PD_SSD
+      dataDiskSizeGb: 10
+      ipConfiguration:
+        ipv4Enabled: true
+```
+
+Save the above as `cloudsql-class.yaml` and apply it to enable dynamic
+provisioning of `CloudSqlInstance` managed resources:
+
+```bash
+kubectl apply -f cloudsql-class.yaml
+```
+
+Now you can omit the `resourceRef` when you create resource claims. Save the
+below resource claim as `mysql-dynamic-claim.yaml`:
+
+```yaml
+apiVersion: database.crossplane.io/v1alpha1
+kind: MySQLInstance
+metadata:
+  name: example-mysql-dynamic-claim
+spec:
+  classSelector:
+    matchLabels:
+      guide: getting-started
+  writeConnectionSecretToRef:
+    name: example-mysql-dynamic-connection-details
+```
+
+When you apply this `MySQLInstance` claim you'll see that it dynamically
+provisions a new `CloudSQLInstance` to satisfy the resource claim:
+
+```bash
+$ kubectl --namespace default apply -f mysql-dynamic-claim.yaml
+mysqlinstance.database.crossplane.io/example-mysql-dynamic-claim created
+
+$ kubectl get mysqlinstance example-mysql-dynamic-claim
+NAME                          STATUS   CLASS-KIND              CLASS-NAME               RESOURCE-KIND      RESOURCE-NAME                               AGE
+example-mysql-dynamic-claim            CloudsqlInstanceClass   example-cloudsql-class   CloudsqlInstance   default-example-mysql-dynamic-claim-bwpzd   47s
+```
+
+You just dynamically provisioned a `CloudsqlInstance`! You can find the name of
+your new `CloudsqlInstance` under the `RESOURCE-NAME` column when you run
+`kubectl describe mysqlinstance`. Reuse the resource class as many times as you
+like; simply submit more `MySQLInstance` resource claims to create more CloudSQL
+instances.
+
+You may have noticed that your resource claim included a `classSelector`. The
+class selector lets you select which resource class to use by [matching its
+labels]. Resource claims like `MySQLInstance` can match different kinds of
+resource class using label selectors, so you could just as easily use the
+exact same `MySQLInstance` to create an Amazon Relational Database Service (RDS)
+instance by creating an `RDSInstanceClass` labelled as `guide: getting-started`.
+When multiple resource classes match the class selector a matching class is
```

Comment:
class selector, a matching c

Reply:
Funnily enough it was intuition to write the sentence that way originally, but I tend to overuse commas and decided to avoid one in this case. I just checked your suggestion using a few grammar apps and the agree with you - fixed.